addons/live-featured-project/addon.json

```diff
@@ -12,6 +12,19 @@
       "name": "Hans5958",
       "note": "remake",
       "id": "remake"
+    },
+    {
+      "name": "ErrorGamer2000",
+      "link": "https://github.com/ErrorGamer2000",
+      "note": "update",
+      "id": "update"
+    }
+  ],
+  "info": [
+    {
+      "id": "privacy",
+      "type": "warning",
+      "text": "Enabling the \"Use Scratch username\" when using the TurboWarp player will give TurboWarp access to your Scratch username."
```

Comment:
The setting for this is already pretty self-explanatory, so does this warning even need to be here?

Reply:
I think we should stay consistent with the TurboWarp player addon. Doesn't make sense to discuss the same things twice